net/http/src/THttpServer.cxx

```diff
@@ -821,20 +824,22 @@ void THttpServer::ProcessRequest(THttpCallArg *arg)
       if (!handler) {
          // for the moment only TCanvas is used for web sockets
          arg->Set404();
-      } else if (arg->fQuery == "connect") {
+      } else if ((arg->fQuery == "connect") || (arg->fQuery == "connect_raw")) {
```

Comment:
For a different PR: if the set of `fQuery` strings is a closed set, please use an enum with a string array in case you need to transform the value into strings. String comparisons to determine a kind is evil.

Reply:
> if the set of fQuery strings is a closed set

fQuery is string, which placed in the URL after "?" mark. Like: 

http://localhost:8080/main.htm?par1=10&par2&item3=[that ever]

In this particular case it is only specific for long-poll socket connection